[2021-08-04 13:25:54,237] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_B 2021-08-04T13:25:49.229724+00:00 [queued]>
[2021-08-04 13:25:54,310] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_B 2021-08-04T13:25:49.229724+00:00 [queued]>
[2021-08-04 13:25:54,310] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 13:25:54,311] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-04 13:25:54,311] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 13:25:54,346] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): training_model_B> on 2021-08-04T13:25:49.229724+00:00
[2021-08-04 13:25:54,374] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'training_model_B', '2021-08-04T13:25:49.229724+00:00', '--job-id', '675', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpbq4uemg3', '--error-file', '/tmp/tmppcn4oyt6']
[2021-08-04 13:25:54,378] {standard_task_runner.py:77} INFO - Job 675: Subtask training_model_B
[2021-08-04 13:25:54,369] {standard_task_runner.py:52} INFO - Started process 8021 to run task
[2021-08-04 13:25:54,659] {logging_mixin.py:104} INFO - Running <TaskInstance: my_dag.training_model_B 2021-08-04T13:25:49.229724+00:00 [running]> on host 04479ea62028
[2021-08-04 13:25:54,939] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=training_model_B
AIRFLOW_CTX_EXECUTION_DATE=2021-08-04T13:25:49.229724+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-04T13:25:49.229724+00:00
[2021-08-04 13:25:54,940] {python.py:151} INFO - Done. Returned value was: 4
[2021-08-04 13:25:55,071] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=training_model_B, execution_date=20210804T132549, start_date=20210804T132554, end_date=20210804T132555
[2021-08-04 13:25:55,171] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-04 13:25:55,198] {local_task_job.py:149} INFO - Task exited with return code 0
